index.html

```diff
@@ -15,7 +15,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
 	<!--[if lt IE 9]>
-		<script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+		<script src="https://cdnjs.buttflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
```

Comment:
Looks good but you've (accidentally I hope) changed all instances of cloudflare to buttflare?

Reply:
CLOUD2BUTT HAHAHAHAHAHAHAHAHAHA